Sources/AppcuesKit/Presentation/Debugger/Panel/DebugFontUI.swift

```diff
@@ -12,14 +12,42 @@ import SwiftUI
 @available(iOS 13.0, *)
 internal enum DebugFontUI {
     struct FontListView: View {
-        let sections: [(title: String, names: [String])]
+        static let sections: [(title: String, names: [String])] = {
+            let familyNames: [String] = (Bundle.main.infoDictionary?["UIAppFonts"] as? [String] ?? [])
```

Comment:
noting that [previously](https://github.com/appcues/appcues-ios-sdk/pull/457) there was an attempt to move the font list calculation out of the view layer - this seems to reverse back. Any downside to this?

Reply:
The previous change ended up being very inefficient since the list was recalculated every time the debug view refreshed. A static property here means the list gets calculated once, when the view appears. I could move the static property somewhere else...maybe an extension on `Font` would make sense.